### PR TITLE
 added input component

### PR DESCRIPTION
**Pull request by varnita619**
- added label input with validation and errors style
- added radio buttons
- added check box buttons

### CODE REVIEW

**Comment by SJTGSHIVAM** [comment]
Changes look clean in the future you can also try to use consistent values for padding and margin.
For now, it looks good You can think and apply this at a later stage.
We are good to merge!

**Review by pritam-kr — APPROVED** [review]
Everything looks good  to go

File: Components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+.radio-section input,.checkbox-section input{
+    cursor: pointer;
+    margin: 0 10px;
+}
+
+.disable{
+    color: var(--disable-button-color);
+}
```
Comment (pranjalmahajan96) [inline Components/input/input.css:71]: add new line at the end of the file
you can also enable addFinalNewLine in vscode settings

File: Components/input/input.html [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+                <i class="fa fa-moon" aria-hidden="true" title="light-dark mode"></i>
+            </div>
+        </div>
+
+
+        <!-- Navbar End -->
+
+        <!-- Side Bar Start -->
+
+        <div class="install-container">
```
Comment (pranjalmahajan96) [inline Components/input/input.html:65]: use proper classnames that define its contents eg. main-container

File: Components/input/input.html [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+                        </li>
+                    </ul>
+                </div>
+
+            </div>
+
+            <!-- Side Bar End -->
+
+            <!-- Badges on Avatar Start -->
+            <main class="component-content-container">
```
Comment (pranjalmahajan96) [inline Components/input/input.html:137]: good use of meta tags

File: Components/input/input.html [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+
+                <div class=" component">
```
Comment (pranjalmahajan96) [inline Components/input/input.html:139]: remove empty space before classname

File: Components/input/input.html [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+
+            </div>
+
+            <!-- Side Bar End -->
+
+            <!-- Badges on Avatar Start -->
```
Comment (pranjalmahajan96) [inline Components/input/input.html:136]: unrelated comment

**Review by pranjalmahajan96 — COMMENTED** [review]
pretty clean PR, needs improvements in only few places

File: Components/input/input.css [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+    color: var(--danger-color);
+}
+
+.radio-title, .checkbox-title{
+    display: flex;
+    align-items: center;
+    margin: 10px;
+    font-size: 13px;
+}
+
```
Comment (mdarif6) [inline Components/input/input.css:63]: You have used rem in all file except few like in line no- 60 and 61- These can be changed in rem too.
Other wise all codes looking good.

**Review by mdarif6 — APPROVED** [review]

File: Components/input/input.css (hunk 6 above)
Comment (varnita619) [reply Components/input/input.css:63]: ok sure

**Pull request merged**